superset/utils/screenshots.py

```diff
@@ -54,6 +56,57 @@
     from flask_caching import Cache
 
 
+class StatusValues(Enum):
+    PENDING = "Pending"
+    COMPUTING = "Computing"
+    UPDATED = "Updated"
+    ERROR = "Error"
+
+
+class ScreenshotCachePayload:
+    def __init__(self, image: bytes | None = None):
+        self._image = image
+        self._timestamp = datetime.now().strftime("%Y/%m/%d-%H:%M:%S")
```

Comment:
Hmm, I'd prefer not to use a custom format here. Let's just use `isoformat()`
```suggestion
        self._timestamp = datetime.now().isoformat()
```